### DISCUSSION TITLE
What controls "No articles found" and footer appearance on front page?

### DISCUSSION THREAD

**Discussion by epogrebnyak** [gethinode/hinode#902] [Help] [answered]
Wonder if there a way to disable to footer entirely and "No articles found." message. 

Should `layout` be edited? Are there parameters for this? Many thanks in advance.
 
```
    [en.params.social]
        title = "" # "Follow me"
        caption = "" # "I work on everything coding and tweet developer memes"
```

**Comment by markdumay** [comment, marked as answer]
You can suppress the *social footer* by creating an empty file in `layouts/partials/footer/social.html`, see the [docs](https://github.com/gethinode/docs/blob/main/layouts/partials/footer/social.html) repo for an example. You can do the same for the *footer*, which is rendered in `layouts/partials/footer/footer.html`.

The `No articles found.` message indicates you do not have any content to be rendered yet. You might want to read up on the [page section configuration](https://gethinode.com/docs/configuration/layout/#page-sections). You can check the[ example site ](https://github.com/gethinode/hinode/tree/main/exampleSite)for an example configuration that displays two sections: `blog` and `projects`.

**Reply by epogrebnyak** [reply]
Thank you!
